src/main/java/com/laserfiche/repository/api/clients/impl/model/PostEntryChildrenRequest.java

```diff
@@ -1,142 +1,21 @@
 package com.laserfiche.repository.api.clients.impl.model;
 
-import com.google.gson.annotations.SerializedName;
-import io.swagger.v3.oas.annotations.media.Schema;
-
-import java.util.Objects;
-/**
- * PostEntryChildrenRequest
- */
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2022-08-19T17:07:43.799-04:00[America/New_York]")
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class PostEntryChildrenRequest {
-  @SerializedName("name")
-  private String name = null;
-
-  @SerializedName("entryType")
-  private PostEntryChildrenEntryType entryType = null;
-
-  @SerializedName("targetId")
-  private Integer targetId = null;
-
-  @SerializedName("sourceId")
-  private Integer sourceId = null;
-
-  public PostEntryChildrenRequest name(String name) {
-    this.name = name;
-    return this;
-  }
-
-   /**
-   * The name of the entry.
-   * @return name
-  **/
-  @Schema(description = "The name of the entry.")
-  public String getName() {
-    return name;
-  }
-
-  public void setName(String name) {
-    this.name = name;
-  }
-
-  public PostEntryChildrenRequest entryType(PostEntryChildrenEntryType entryType) {
-    this.entryType = entryType;
-    return this;
-  }
-
-   /**
-   * The type of the entry.
-   * @return entryType
-  **/
-  @Schema(description = "The type of the entry.")
-  public PostEntryChildrenEntryType getEntryType() {
-    return entryType;
-  }
-
-  public void setEntryType(PostEntryChildrenEntryType entryType) {
```

Comment:
It depends. So, if it's a public field and if access (getter) and mutation (setter) don't require extra operations, it's okay to do so via dot nodation. This is a compromise for the class to work with Boomi. Because if it contains private field, druing de/serialization, it will violate security rules.